fmlcore/src/main/java/net/minecraftforge/fml/ModContainer.java

```diff
@@ -174,6 +176,10 @@ public void dispatchConfigEvent(IConfigEvent event) {
         }
     }
 
+    public @Nullable IEventBus getEventBus() {
```

Comment:
```suggestion
    public @Nullable IEventBus getModEventBus() {
```
Considering that the only usage of this is in `FMLConstructModEvent` for grabbing the mod event bus, renaming the method accordingly would be clearer and match the name of the (now deprecated) `FMLJavaModLoadingContext.getModEventBus()` method